src/malloy/ipython/tab_renderer.py

```diff
@@ -143,8 +161,7 @@
     </div>
   </div>
   <div class="result-middle tabset-{rand}" data-result-kind="html" id="HTML-{rand}">
-    <div class="result-inner">
-      {html}
+    <div class="result-inner" id="HTML-{rand}-inner">
```

Comment:
Why not just make this `id="{result_div}"` and use result_div in the format instead of creating the same id in two different places, I could see that causing issues in the future if one was modified by not the other.

Reply:
ya makes sense.